engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -1,316 +1,459 @@
+  "nui-editor-disable-autosave": "Vypnout automatické ukládání",
+  "nui-editor-disable-icons": "Vypnout ikony v náhledu stromu",
+  "nui-editor-layout": "Zarovnat",
+  "nui-editor-new": "Nový",
+  "nui-editor-override": "Uložit",
+  "nui-editor-paste": "Vložit",
+  "nui-editor-redo": "Opakovat",
+  "nui-editor-save": "Uložit jako...",
+  "nui-editor-settings-title": "Nastavení editoru",
+  "nui-editor-undo": "Vrátit",
+  "oculus-rift-support": "Podpora Oculus Rift",
+  "online-players": "Online hráči",
+  "open-inventory": "Otevřít inventář",
+  "open-in-browser": "Otevřít v prohlížeči",
+  "outline": "Okraje",
+  "parallax-mapping": "Paralaxní mapování",
+  "parsing-content": "Zpracování obsahu ...",
+  "particle-effect-limit": "Limit částicových efektů",
+  "ping": "Odezva",
+  "player-color": "Barva hráče",
+  "player-eye-height": "Výška pohledu hráče",
+  "player-height": "Výška hráče",
+  "player-name": "Jméno hráče",
+  "player-settings": "Hráč",
+  "player-settings-identities-export": "Exportovat...",
+  "player-settings-identities-import": "Importovat...",
+  "player-settings-title": "Nastavení hráče",
+  "please-wait": "Prosím počkej...",
+  "pregeneration-description": "Vyber svět ze seznamu k náhledu. Klikni na Přegenerovat k náhodnému výběru nového seedu nebo na Nastavit pro detailní úpravu světa",
+  "preview-world-title": "Náhled světa...",
+  "preview-zoom-factor": "Přiblížení",
+  "previous-toolbar-item": "Předchozí předmět v toolbaru",
+  "remove-confirmation-popup-title": "Potvrzení odstranění",
+  "remove-confirmation-popup-message": "Jsi si jistý smazáním?",
+  "remove-binding": "Odebrat",
+  "remove-server": "Odebrat",
+  "rendering-settings": "Konfigurace vykreslovacích modulů",
+  "reset-default": "Resetovat",
+  "re-roll": "Přegenerovat",
+  "reset-fov-amount": "Obnovit zorné pole",
+  "resolution-scale": "Škála rozlišení",
+  "ui-scale": "Škála UI",
+  "respawn": "Oživit",
+  "restore-defaults": "Obnovit výchozí",
+  "return-advanced-setup": "Zpět k Pokročilému nastavení",
+  "return-basic-setup": "Zpět k Základnímu nastavení",
+  "return-main-menu": "Odejít do hlavního menu",
+  "return-pregeneration": "Zpět k Předgeneraci",
+  "return-universe-setup": "Zpět k Nastavení Vesmíru",
+  "return": "Zpět",
+  "rotation-dead-zone": "Vůle osy rotace",
+  "save-game-path": "Cesta k uloženým hrám:",
+  "save-game-details": "Detaily",
+  "screenshot-format": "Formát screenshotu",
+  "screenshot-size": "Velikost screenshotu",
+  "screenshot-size-double": "Dvojitá velikost",
+  "screenshot-size-half": "Poloviční velikost",
+  "screenshot-size-normal": "Normální velikost",
+  "screenshot-size-quarter": "Čtvrtinová velikost",
+  "select-game-title": "Vyber hru",
+  "select-modules": "Moduly...",
+  "select-modules-title": "Výběr Modulů...",
+  "select-modules-filter-reset": "Resetovat filtr kategorií",
+  "select-modules-filter-title": "Kategorie modulu",
+  "select-multiplayer-game-sub-title": "Multiplayer",
```

Comment:
```suggestion
  "select-multiplayer-game-sub-title": "Hra pro více hráčů",
```

Reply:
Again, not sure about translating this, bud older translations have this translated, so let's stick to that